versions.props

```diff
@@ -10,6 +10,7 @@ org.apache.pig:pig = 0.14.0
 com.fasterxml.jackson.*:* = 2.11.4
 com.google.errorprone:error_prone_annotations = 2.3.3
 com.google.guava:guava = 31.1-jre
+com.google.guava:guava-testlib = 31.1-jre
```

Comment:
nit: you should be able to replace the previous line with `com.google.guava:* = 31.1-jre`

Reply:
Done.